internal/storage/repository.go

```diff
@@ -22,3 +26,19 @@ type URLData struct {
 	URL string
 	ID  string
 }
+
+type ReqBatchURLs []ReqBatchURL
```

Comment:
CHANGE REQUEST: it should be in the `model` or `dto` packages.

Reply:
done